rotkehlchen/chain/evm/decoding/uniswap/v3/decoder.py

```diff
@@ -531,7 +531,7 @@ def _maybe_enrich_liquidity_pool_creation(
     ) -> TransferEnrichmentOutput:
         """This method enriches Uniswap V3 LP creation transactions."""
         if (
-            context.event.asset == self.uniswap_v3_nft and
+            self.uniswap_v3_nft in context.event.asset.identifier and  # asset identifier will also include the token id  # noqa: E501
```

Comment:
This is hard to follow because there is so much implicit knowledge here 🙈 

I would create a function where we have all the `evm_address_to_identifier` functions with

```
def identifier_belongs_to_collection(asset_identifier, collection_identifier):
      # some good docstring explaining that the initial part is the same and the only difference is the nft id
      return asset_identifier.startswith(collection_identifier)
```